docs/connectors/mysql.asciidoc

```diff
@@ -895,7 +895,52 @@ Other data type mappings are described in the following sections.
 [[temporal-values]]
 ==== Temporal values
 
-Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend on the value of the `time.precision.mode` configuration property. When `time.precision.mode` configuration property is set to `adaptive` (the default), then the connector will determine the literal type and semantic type for the temporal types based on the column's data type definition so that events _exactly_ represents the values in the database:
+Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend on the value of the `time.precision.mode` configuration property.
+
+[NOTE]
+====
+As of Debezium 0.7 `adaptive_time_microseconds` mode was introduced and is the default `time.precision.mode` for the MySQL connector. Mode `adaptive` was marked as deprecated.
+====
+
+[WARNING]
+====
+Debezium can only capture positive TIME field values in the range of `00:00:00.000000` to `23:59:59.999999` when `time.precision.mode` is set to `adaptive`, and `00:00:00.000` to `23:59:59.999` when `time.precision.mode` is set to `connect`.
+
+MySQL TIME fields value range is `-838:59:59.000000` to `838:59:59.000000`. This range can only be captured when `time.precision.mode` is set to `adaptive_time_microseconds`, which is available as of Debezium 0.7 .
+====
+
+When `time.precision.mode` configuration property is set to `adaptive_time_microseconds` (the default), then the connector will determine the literal type and semantic type for the temporal types based on the column's data type definition so that events _exactly_ represents the values in the database, except that all TIME fields will be captured as microseconds:
+
+[cols="15%a,15%a,35%a,35%a",width=100,options="header,footer",role="table table-bordered table-striped"]
+|=======================
+|MySQL Data Type
+|Literal type (schema type)
+|Semantic type (schema name)
+|Notes
+
+|`DATE`
+|`INT32`
+|`io.debezium.time.Date`
+| Represents the number of days since epoch.
+
+|`TIME[(M)]`
+|`INT64`
+|`io.debezium.time.MicroTime`
+| Represents the time value in microseconds and does not include timezone information. MySQL allows `M` to be in the range 0-6 to store up to microsecond precision.
+
+|`DATETIME`, `DATETIME(0)`, `DATETIME(1)`, `DATETIME(2)`, `DATETIME(3)`
+|`INT64`
+|`io.debezium.time.Timestamp`
+| Represents the number of milliseconds past epoch, and does not include timezone information.
+
+|`DATETIME(4)`, `DATETIME(5)`, `DATETIME(6)`
+|`INT64`
+|`io.debezium.time.MicroTimestamp`
+| Represents the number of microseconds past epoch, and does not include timezone information.
+
+|=======================
+
+When `time.precision.mode` configuration property is set to `adaptive` (deprecated), then the connector will determine the literal type and semantic type for the temporal types based on the column's data type definition so that events _exactly_ represents the values in the database:
```

Comment:
s/When/When the/
s/represents/represent/

Reply:
s/todo/done/

Reply:
also changed all other occurrences according to s/When `time.precision.mode`/When the `time.precision.mode`/